.gitignore

```diff
@@ -926,3 +932,4 @@ lib/cpluff/stamp-h1
 /addons/library.xbmc.pvr/libXBMC_pvr.lib
 
 /pvr-addons
+/adsp-addons
```

Comment:
this isn't needed. `/pvr-addons` was for the old xbmc-pvr-addons repo, where all the add-ons were combined in a single repo. The build system knew to look in the `/pvr-addons` folder for these add-ons.